elide-datastore/elide-datastore-hibernate/src/main/java/com/yahoo/elide/core/hibernate/hql/AbstractHQLQueryBuilder.java

```diff
@@ -88,16 +86,6 @@ public AbstractHQLQueryBuilder withPossibleFilterExpression(Optional<FilterExpre
         return this;
     }
 
-    public AbstractHQLQueryBuilder withPossibleSorting(final Optional<Sorting> possibleSorting) {
```

Comment:
The entity projection includes the Filter as well (not just the sorting and the pagination)